web/src/components/TagTree.tsx

```diff
@@ -6,11 +6,12 @@ import { useMemoFilterStore } from "@/store/v1";
 interface Tag {
   key: string;
   text: string;
+  amount: string;
```

Comment:
number?

Reply:
I can see how this naming is misleading, its a string because it contains the formatted number text that contains the parentheses around the number. I've updated the name to be more descriptive

Reply:
It's better to use number instead of formated string. For example: amount > 1 then show amount string.

Reply:
Move in https://github.com/usememos/memos/pull/3970/commits/e2c96a39b77ae01bbde0d22ee11b041cf9d406c5